plugins/lang/python/python_loader.cc

```diff
@@ -443,9 +443,13 @@ runPythonScript(std::shared_ptr<PythonContext> titl,
     // if we're simply executing the body of the script
     try {
         if(elementToRun == PackageElement::MAIN) {
-                JML_TRACE_EXCEPTIONS(false);
-                boost::python::object obj
-                    = boost::python::exec(scriptSourceStr, pyControl.main_namespace);
+            JML_TRACE_EXCEPTIONS(false);
+            int argc = 1;
```

Comment:
Are you sure argc shouldn't be equal to 1? It is supposed to be the count of strings pointed to by argv. Argv holds one string hence argc should be 1 no?